.github/workflows/release-drafter.yml

```
@@ -25,11 +21,12 @@ jobs:
   update_release_draft:
     permissions:
       contents: write
-      pull-requests: write
+      pull-requests: read
     runs-on: ubuntu-latest
     steps:
       - uses: release-drafter/release-drafter@v7
         with:
-          publish: false  # Always create as draft, publish manually after review
+          publish: false
+          disable-autolabeler: true
```

actionlint (1.7.12)

[error] input "disable-autolabeler" is not defined in action "release-drafter/release-drafter@v7". available inputs are "commitish", "config-name", "dry-run", "filter-by-range", "footer", "header", "include-pre-releases", "initial-commits-since", "latest", "name", "prerelease", "prerelease-identifier", "publish", "tag", "token", "version"

(action)